media-source-respec.html

```diff
@@ -344,11 +344,8 @@ <h2><dfn>MediaSource</dfn> Object</h2>
     undefined             clearLiveSeekableRange ();
     static boolean        isTypeSupported (DOMString type);
 };</pre>
-      <section>
-        <h3>Attributes</h3>
-        <dl class="attributes" data-dfn-for="MediaSource">
 
-          <dt><dfn><code>handle</code></dfn> of type <span class="idlAttrType"><a>MediaSourceHandle</a></span>, [SameObject] readonly</dt><dd>
+          <h3><dfn><code>handle</code></dfn> attribute</h3>
```

Comment:
The outline raises MediaSource's algorithms to the same level as other interfaces. This seems incorrect to me. If the idea was to more precisely enumerate all of the various attributes and methods in the TOC, then consulting how similar was done for Algorithms (within each interface) seems a better route. Something (unproofed) like:

```suggestion
     <section>
        <h3>Attributes</h3>
        <dl class="attributes" data-dfn-for="MediaSource">
          <h4><dfn><code>handle</code></dfn> attribute</h3>
        ....
```

Reply:
Definitely want to get rid of all the `<dl>`s... let me check what happened with the algorithm level.